x/wasm/keeper/querier_test.go

```diff
@@ -708,17 +722,22 @@ func TestQueryPinnedCodes(t *testing.T) {
 			expCodeIDs: []uint64{exampleContract2.CodeID},
 		},
 	}
-	for msg, spec := range specs {
-		t.Run(msg, func(t *testing.T) {
+	for testName, spec := range specs {
+		t.Run(testName, func(t *testing.T) {
 			got, err := q.PinnedCodes(sdk.WrapSDKContext(ctx), spec.srcQuery)
-			if spec.expErr != nil {
-				assert.Nil(t, got)
-				assert.NotNil(t, err)
 
+			if testName == "req nil" || testName == "with invalid pagination key" {
+				assert.Nil(t, got)
+				assert.EqualError(t, spec.expErr, err.Error())
 				return
 			}
```

Comment:
```suggestion
			if spec.expErr != nil {
				assert.Nil(t, got)
				assert.EqualError(t, spec.expErr, err.Error())
			} else {
			}
```

Reply:
I think we need to standardize the use of assertion errors, but that will be implemented in other PRs.